api_sentry.py

```diff
@@ -0,0 +1,199 @@
+    # ?limit=10&query=is:for_review&sort=freq&statsPeriod=1d
+    def issues(self, release_version):
+        return self.client.get(
+            (
+                'organizations/{0}/issues/'
+                '?project={1}'
+                '&query=is:for_review release.version:{2}'
+                '&sort=freq&statsPeriod=1d'
+            ).format(self.organization_slug, self.project_id, release_version)
+        )
+
+    # API: Releases
+    # The most recent releases of the project:
+    # projects/mozilla/firefox-ios/releases/
+    # ?per_page=100&project=1111111111
+    # &statsPeriod=1d&environment=Production
+    def releases(self):
+        return self.client.get(
+            (
+                'projects/{0}/firefox-ios/releases/'
+                '?&project={1}&statsPeriod=1d'
+                '&environment=Production'
+            ).format(self.organization_slug, self.project_id)
+        )
+
+
+class SentryClient(Sentry):
+
+    def __init__(self):
+        print("SentryClient.__init__()")
+        super().__init__()
+        self.db = DatabaseSentry()
+
+    def data_pump():
+        # Let's leave this to stay consistent with other
+        # api_*.py files.
+        pass
+
+    def sentry_releases(self):
+        print("SentryClient.sentry_releases()")
+        releases = self.releases()
+        release_versions = self.db.report_version_strings(releases)
+        return release_versions
+
+    def sentry_issues(self):
+        print("SentryClient.sentry_issues()")
+
+        release_versions = self.sentry_releases()
+
+        df_issues = pd.DataFrame()
+        for release_version in release_versions:
+            issues = self.issues(release_version)
+            # NOTE: Use just the last two major releases for now
+            df_issues_release = self.db.report_issue_payload(issues,
+                                                             release_version)
+            # output CSV for debugging
+            df_issues_release.to_csv(
+                "sentry_issues_{0}.csv".format(release_version),
+                index=False)
+
+            # Insert issues from this release into the same dataframe
+            df_issues = pd.concat([df_issues, df_issues_release], axis=0)
+
+        # Insert into database
+        self.db.issue_insert(df_issues)
+
+
+class DatabaseSentry():
+
+    def __init__(self):
+        print("DatabaseSentry.__init__()")
+        super().__init__()
+        self.db = Database()
+
+    def _is_version_numeric(self, version):
+        version = version.strip()
+        if version is None or version == '' or version == '9000':
+            return False
+        if "(" in version or ")" in version:
+            return False
+        if "org.mozilla.ios.Firefox" in version:
+            return False
+        parts = version.split('.')
+        return all(p.isdigit() for p in parts) and len(parts) > 0
+
+    def _get_major_versions(self, versions):
+        NUM_MAJOR_VERSIONS = 10
```

Comment:
Let's look for unassigned issues for the last 10 major versions. Usually the issues concentrate in the last 2 major versions.

Reply:
NUM_MAJOR_VERSIONS is now `2`: One for beta and one for release.